### PR TITLE
Add BCD for Gamepad features related to trigger-rumble haptic effect

### PR DESCRIPTION
**Pull request by chrisdavidmills**
#### Summary

The Gamepad API [`"trigger-rumble"`](https://w3c.github.io/gamepad/#dom-gamepadhapticeffecttype-trigger-rumble) haptic effect has been supported since Chrome 126 (see the [ChromeStatus entry](https://chromestatus.com/feature/5162940951953408)).

This PR adds BCD for related features:

- `"trigger-rumble"` as an effect type that can be used in the `GampadHapticActuator.effects` property.
- `"trigger-rumble"` as an effect type that can be used in the `GampadHapticActuator.playEffect()` method. 
- `leftTrigger` and `rightTrigger` as params that can be used in the `GampadHapticActuator.playEffect()` method.

### CODE REVIEW

File: api/GamepadHapticActuator.json [hunk 1]
```diff
@@ -143,6 +143,40 @@
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "trigger_rumble": {
```
Comment (queengooborg) [inline api/GamepadHapticActuator.json:147]:
```suggestion
        "type_trigger-rumble": {
```

File: api/GamepadHapticActuator.json [hunk 2]
```diff
@@ -181,6 +215,74 @@
...
```
Comment (queengooborg) [inline api/GamepadHapticActuator.json:219]:
```suggestion
        "type_trigger-rumble": {
```

File: api/GamepadHapticActuator.json [hunk 3]
```diff
@@ -181,6 +215,74 @@
...
+            "status": {
+              "experimental": true,
+              "standard_track": true,
+              "deprecated": false
+            }
+          }
+        },
+        "trigger_rumble_params": {
+          "__compat": {
+            "description": "<code>leftTrigger</code> and <code>rightTrigger</code> params",
```
Comment (queengooborg) [inline api/GamepadHapticActuator.json:255]:
```suggestion
        "params_leftTrigger_parameter": {
          "__compat": {
            "description": "<code>params.leftTrigger</code> parameter",
```

File: api/GamepadHapticActuator.json [hunk 4]
```diff
@@ -181,6 +215,74 @@
...
+              "samsunginternet_android": "mirror",
+              "webview_android": "mirror"
+            },
+            "status": {
+              "experimental": true,
+              "standard_track": true,
+              "deprecated": false
+            }
+          }
         }
```
Comment (queengooborg) [inline api/GamepadHapticActuator.json:286]:
```suggestion
        },
        "params_rightTrigger_parameter": {
          "__compat": {
            "description": "<code>params.rightTrigger</code> parameter",
            "spec_url": "https://w3c.github.io/gamepad/#dom-gamepadeffectparameters-righttrigger",
            "support": {
              "chrome": {
                "version_added": "126"
              },
              "chrome_android": "mirror",
              "edge": "mirror",
              "firefox": {
                "version_added": false
              },
              "firefox_android": "mirror",
              "ie": {
                "version_added": false
              },
              "oculus": "mirror",
              "opera": "mirror",
              "opera_android": "mirror",
              "safari": {
                "version_added": false
              },
              "safari_ios": "mirror",
              "samsunginternet_android": "mirror",
              "webview_android": "mirror"
            },
            "status": {
              "experimental": true,
              "standard_track": true,
              "deprecated": false
            }
          }
```

**Review by queengooborg — CHANGES_REQUESTED** [review]
Some minor changes to the feature names, as well as separating the new parameter feature into individual parameters for machine readability!

**Comment by chrisdavidmills** [comment]
Your comments all sound pretty reasonable, so I've accepted them all. Thanks @queengooborg!

**Comment by chrisdavidmills** [comment]
I also just marked the `type` property as deprecated, on the advice of the Google engineering folks. It no longer appears in the spec, so it makes sense.

**Review by queengooborg — APPROVED** [review]

**Pull request merged**